### ISSUE TITLE
Blazor Hybrid + Web solution template missing project refs in .sln when created via VS Code

### ISSUE THREAD

**Issue by BethMassi** [dotnet/maui#22381]
### Description

Trying out the new .NET Blazor Hybrid and Web App template in VS Code (with C# Dev Kit and MAUI extension installed), the .SLN file isn't generated properly so the solution never opens in the Solution Explorer. This happens on Windows and Mac. 

### Steps to Reproduce

C# Dev Kit v1.5.20
.NET MAUI extension v0.11.87 
Microsoft.Maui.Templates.net8.8.0.100-dev.nupkg installed
Windows or Mac

- Create a new .NET project in VS Code (Windows or Mac) via the command pallet and select the new .NET MAUI Blazor and Web App template. 

![VSCodeCreate](https://github.com/dotnet/maui/assets/5115571/6a036c2b-7b4d-4fd4-812c-f0141a3352da)

- Notice that it doesn't open the project in VS Code
- Open the folder and the projects are created but the .SLN is missing the project references. 

```code
Microsoft Visual Studio Solution File, Format Version 12.00
# Visual Studio Version 17
VisualStudioVersion = 17.0.31903.59
MinimumVisualStudioVersion = 10.0.40219.1
Global
	GlobalSection(SolutionConfigurationPlatforms) = preSolution
		Debug|Any CPU = Debug|Any CPU
		Release|Any CPU = Release|Any CPU
	EndGlobalSection
	GlobalSection(SolutionProperties) = preSolution
		HideSolutionNode = FALSE
	EndGlobalSection
EndGlobal
```

### Version with bug

8.0.10 SR3

### Is this a regression from previous behavior?

No, this is something new

### Last version that worked well

Unknown/Other

### Affected platforms

Windows, macOS

**Comment by Zhanglirong-Winnie** [comment]
Verified this issue with VSCode 1.89.1(.net9). Can repro this issue. 
![image](https://github.com/dotnet/maui/assets/77434865/94a94461-7304-4bb6-b100-c9b7fd206eaa)

**Comment by BethMassi** [comment]
This is still happening in VSCode v1.92 with C# DevKit v1.11.6 and .NET MAUI extension v1.3.29

**Comment by mkArtakMSFT** [comment]
@vijayrkn do you own project creation experience in VSCode? Looks like the project creation for VS Code only is broken for this solution, and it works just fine when created from the CLI.

**Comment by vijayrkn** [comment]
Hey @mkArtakMSFT Tagging @smitpatel @JakeRadMSFT who owns this experience.

**Comment by smitpatel** [comment]
Where is the source for template?

**Comment by BethMassi** [comment]
@smitpatel https://github.com/dotnet/maui/tree/net9.0/src/Templates/src/templates/maui-blazor-solution

**Comment by smitpatel** [comment]
https://github.com/dotnet/maui/blob/0f4a170cfd3af55770ade65d3e5145c609c4abb9/src/Templates/src/templates/maui-blazor-solution/.template.config/template.json#L25-L40

A project or solution template should add paths to csproj files in primary outputs. (the condition for dotnetcli is incorrect.

**Comment by BethMassi** [comment]
@smitpatel cool, is this something you can fix? Or @Eilon ?

**Comment by smitpatel** [comment]
@BethMassi submitted #25303

**Comment by Eilon** [comment]
Fixed by #25303. Thanks @smitpatel !!
